### PR TITLE
update customized algos doc

### PR DESCRIPTION
**Pull request by J-shang**
Add how to migrate previous customized algorithms.

### CODE REVIEW

**Review by liuzhe-lz — APPROVED** [review]

File: docs/en_US/Tutorial/InstallCustomizedAlgos.rst [hunk 1]
```diff
@@ -160,3 +160,9 @@ Run following command to uninstall an installed package:
 For example:
 
 ``nnictl algo unregister demotuner``
+
+
+Migrate previous customized algorithms
```
Comment (QuanluZhang) [inline docs/en_US/Tutorial/InstallCustomizedAlgos.rst:165]: Porting customized algorithm worked on v1.X to v2.X

Comment (J-shang) [reply docs/en_US/Tutorial/InstallCustomizedAlgos.rst]: fix it

File: docs/en_US/Tutorial/InstallCustomizedAlgos.rst [hunk 2]
```diff
@@ -2,6 +2,16 @@
...
 
+Content
+-------
+
+* `Overview`_
+* `Register customized algorithms as builtin tuners, assessors and advisors`_
+* `Use the installed builtin algorithms in experiment`_
+* `Manage builtin algorithms using nnictl algo`_
+* `Porting customized algorithms from v1.x to v2.x`_
+* `Example: Register a customized tuner as a builtin tuner`_
```
Comment (QuanluZhang) [inline docs/en_US/Tutorial/InstallCustomizedAlgos.rst:13]: no need to manually write this, you can simply use `.. contents::`

File: docs/en_US/Tutorial/InstallCustomizedAlgos.rst [hunk 3]
```diff
@@ -103,10 +113,10 @@ Run following command to register the customized algorithms as builtin algorithm
 
 The ``<path_to_meta_file>`` is the path to the yaml file your created in above section.
 
-Reference `customized tuner example <../Tuner/InstallCustomizedTuner.rst>`_ for a full example.
+Reference `Example: Register a customized tuner as a builtin tuner`_ for a full example.
```
Comment (QuanluZhang) [inline docs/en_US/Tutorial/InstallCustomizedAlgos.rst:116]: use reference link here

File: docs/en_US/Tutorial/InstallCustomizedAlgos.rst (hunk 2 above)
Comment (J-shang) [reply docs/en_US/Tutorial/InstallCustomizedAlgos.rst]: fix it

File: docs/en_US/Tutorial/InstallCustomizedAlgos.rst (hunk 3 above)
Comment (J-shang) [reply docs/en_US/Tutorial/InstallCustomizedAlgos.rst]: fix it

**Review by QuanluZhang — APPROVED** [review]

**Pull request merged**